src/generated/java/com/prowidesoftware/swift/model/field/FieldEnum.java

```diff
@@ -0,0 +1,738 @@
+    F70E,
+    F70F,
+    F70G,
+    F70H,
+    F70,
+    F71A,
+    F71B,
+    F71C,
+    F71D,
+    F71E,
+    F71F,
+    F71G,
+    F71M,
+    F71N,
+    F72Z,
+    F72,
+    F73A,
+    F73R,
+    F73S,
+    F73,
+    F74,
+    F75,
+    F76,
+    F77A,
+    F77B,
+    F77C,
+    F77D,
+    F77E,
+    F77F,
+    F77H,
+    F77J,
+    F77L,
+    F77T,
+    F77U,
+    F77,
+    F78B,
+    F78,
+    F78D,
+    F79Z,
+    F79,
+    F80C,
+    F81A,
+    F81D,
+    F81J,
+    F82A,
+    F82B,
+    F82D,
+    F82J,
+    F83A,
+    F83C,
+    F83D,
+    F83J,
+    F84A,
+    F84B,
+    F84D,
+    F84J,
+    F85A,
+    F85B,
+    F85D,
+    F85J,
+    F86A,
+    F86B,
+    F86D,
+    F86J,
+    F86,
+    F87A,
+    F87B,
+    F87D,
+    F87J,
+    F88A,
+    F88B,
+    F88D,
+    F88J,
+    F89A,
+    F89D,
+    F89J,
+    F90A,
+    F90B,
+    F90C,
+    F90D,
+    F90E,
+    F90F,
+    F90J,
+    F90K,
+    F90L,
+    F91A,
+    F91D,
+    F91J,
+    F92A,
+    F92B,
+    F92C,
+    F92D,
+    F92F,
+    F92H,
+    F92J,
+    F92K,
+    F92L,
+    F92M,
+    F92N,
+    F92P,
+    F92R,
+    F93A,
+    F93B,
+    F93C,
+    F93D,
+    F93E,
+    F93F,
+    F94A,
+    F94B,
+    F94C,
+    F94D,
+    F94E,
+    F94F,
+    F94G,
+    F94H,
+    F94L,
+    F95C,
+    F95L,
+    F95P,
+    F95Q,
+    F95R,
+    F95S,
+    F95U,
+    F95V,
+    F96A,
+    F96D,
+    F96J,
+    F97A,
+    F97B,
+    F97C,
+    F97D,
+    F97E,
+    F98A,
+    F98B,
+    F98C,
+    F98D,
+    F98E,
+    F98F,
+    F98G,
+    F98H,
+    F98J,
+    F98K,
+    F99A,
+    F99B,
+    F99C,
+    F101,
+    F102,
+    F103,
+    F104,
+    F105,
+    F106,
+    F107,
+    F108,
+    F109,
+    F110,
+    F111,
+    F113,
+    F114,
+    F115,
+    F116,
+    F117,
+    F118,
+    F119,
+    F120,
+    F121,
+    F122,
+    F123,
+    F124,
+    F125,
+    F126,
+    F127,
+    F128,
+    F129,
+    F130,
+    F132,
+    F133,
+    F134,
+    F135,
+    F136,
+    F137,
+    F140,
+    F141,
+    F142,
+    F143,
+    F144,
+    F151,
+    F152,
+    F153,
+    F165,
+    F171,
+    F172,
+    F173,
+    F174,
+    F175,
+    F177,
+    F179,
+    F199,
+    F202,
+    F203,
+    F204,
+    F206,
+    F208,
+    F209,
+    F242,
+    F243,
+    F251,
+    F252,
+    F253,
+    F254,
+    F255,
+    F256,
+    F257,
+    F258,
+    F259,
+    F260,
+    F263,
+    F264,
+    F270,
+    F271,
+    F272,
+    F280,
+    F281,
+    F301,
+    F302,
+    F303,
+    F304,
+    F305,
+    F307,
+    F311,
+    F312,
+    F313,
+    F329,
+    F330,
+    F331,
+    F332,
+    F333,
+    F334,
+    F335,
+    F336,
+    F337,
+    F338,
+    F339,
+    F340,
+    F341,
+    F342,
+    F343,
+    F344,
+    F345,
+    F346,
+    F347,
+    F348,
+    F349,
+    F401,
+    F405,
+    F421,
+    F422,
+    F423,
+    F424,
+    F425,
+    F431,
+    F432,
+    F433,
+    F434,
+    F443,
+    F451,
+    F461,
+    F503,
+    F533,
+    F619,
+    F999;
+
+    public String getFieldName() {
```

Comment:
for convention when method is not an accessor we do not add the "get" prefix. This should be fieldName()

Reply:
Ok